cloud_guardian/iam_model/graph/permission/actions.py

```diff
@@ -102,13 +139,34 @@ class ActionFactory:
         PutObject.aws_action: PutObject,
         Any.aws_action: Any,
         AssumeRole.aws_action: AssumeRole,
+        # NOTE: should all actions be defined and listed in the code?
```

Comment:
only those that we support as we think are relevant for priviledge escalation attacks, we should ignore other actions

Reply:
Ok. In our example, do you think we should ignore some of them (`DevS3Access` or `ReadOnly`)?